s3prl/downstream/README.md

```diff
@@ -227,20 +246,37 @@ Specified by the command `-d asr`
         datarc:
             libri_root: "root directory of LibriSpeech"
     ```
-
 4. Prepare the lengths for utterances in LibriSpeech's train-clean-100, dev-clean and test-clean:
 
     ```bash
     # Official LibriSpeech is in .flac format
     python3 preprocess/generate_len_for_bucket.py -i "root directory of LibriSpeech" -o data/librispeech -a .flac --n_jobs 12
     ```
 
+
 #### Training
 
+If you're running on the `master` branch of `s3prl` you can simply run the followign to train:
+
 ```bash
 python3 run_downstream.py -n ExpName -m train -u fbank -d asr
 ```
 
+If you're working on the `huggingface2` branch, you first need to edit the following fields to `downstream_expert.datatrc` subset of `config.yaml`:
```

Comment:
Since now we fix the asr config to have `dict_path` as default, I guess we don't need to add this part of README right?